generic/tests/cfg/edk2_stability_test.cfg

```diff
@@ -0,0 +1,9 @@
+- edk2_stability_test:
+    virt_test_type = qemu libvirt
```

Comment:
Move to `qemu/tests/cfg` and remove `libvirt`

Reply:
Corrected